README.md

```diff
@@ -5,13 +5,44 @@ Run an open-source project? Submit a Pull Request to add yourself to the list!
 
 Visit the website: [up-for-grabs.net](http://up-for-grabs.net/)
 
+## Add your project
+
+Each of the projects is a file in the [projects](https://github.com/dahlbyk/up-for-grabs.net/blob/gh-pages/_data/projects/) folder - to add yours, create a new file named after your project. Ensure all spaces and special characters are replaced with `-`, to make everyone's life easier.
```

Comment:
Maybe add a small remark that this can be done through GitHub's web UI rather than having to clone the repo.